src/pages/SignUpResearcher.js

```diff
@@ -44,13 +51,22 @@ function SignUpResearcher() {
     return errors;
   };
 
-  const handleSubmit = (values) => {
-    //TODO: authenticate to allow login
-
-    // Db.addResearchData(useContext(UserContext)?.uid, ...); TODO
-    // navigate to next page
-
+  const handleSubmit = async values => {
+    console.log(values);
+    try {
+      await Db.addResearchData(userId, values).then;
```

Comment:
just a quick question, what does the `.then` here do? i'm a bit rusty on js, but i rarely see a dangling `.then`

Reply:
Good catch Wenny! Removed it, I don't think it is necessary (am a js noob and went to google, did not find a dangling .then question on SO)

Reply:
Sorry, my bad!! I accidentally left it in for my previous db function 😅, just removed it